packages/codex-harness/scripts/codex_review.py

```
@@ -0,0 +1,244 @@
+    )
+    parser.add_argument("--base", default="main", help="Base branch/ref (default: main)")
+    parser.add_argument("--project", default=".", help="Project root (default: cwd)")
+    parser.add_argument("--allow-untrusted-hooks", action="store_true")
+    parser.add_argument("--timeout", type=int, default=DEFAULT_TIMEOUT_SECONDS)
+    return parser.parse_args(argv)
+
+
+def run_git(repo_root: Path, args: list[str], timeout: int = 30) -> str:
+    """Run a git subcommand and return raw stdout (empty string on error)."""
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)